src/components/Steps/Steps.tsx

```diff
@@ -63,6 +66,7 @@ const Steps: VibeComponent<StepsProps> & { types?: typeof StepsType } = forwardR
       >
         {isContentOnTop && steps[activeStepIndex]}
         <StepsHeader
+          className={headerWrapperClassName}
```

Comment:
we need a good reason to support this... why?

Reply:
Not needed eventually 👍